tools/postag/src/main/scala/org/allenai/nlpstack/postag/StanfordPostagger.scala

```diff
@@ -0,0 +1,43 @@
+package org.allenai.nlpstack.postag
+
+import edu.stanford.nlp.ling.CoreLabel
+import edu.stanford.nlp.tagger.maxent.MaxentTagger
+
+import org.allenai.nlpstack.core._
+import org.allenai.datastore.Datastore
+
+import java.net.URL
+import scala.collection.JavaConversions._
+
+class StanfordPostagger(
+    val tagger: MaxentTagger
+) extends Postagger {
+
+  def this() = this(StanfordPostagger.loadDefaultModel())
+
+  override def postagTokenized(tokens: Seq[Token]): Seq[PostaggedToken] = {
+    val postags = tagger.tagSentence(
+      tokens.map { token =>
+      val corelabel = new CoreLabel();
+      corelabel.setWord(token.string);
+      corelabel
+    }.toList
+    ).map(_.tag())
+
+    (tokens zip postags) map {
+      case (token, postag) =>
+        PostaggedToken(token, postag)
+    }
+  }
+}
+
+object StanfordPostagger {
+  def loadDefaultModel(): MaxentTagger = {
+    val filePath = Datastore.directoryPath(
+      "edu.stanford.nlp.models.pos-tagger",
+      "english-left3words-3.4.1",
+      1
+    )
+    new MaxentTagger(filePath.toString + "/english-left3words-distsim.tagger")
```

Comment:
Should be: 

new MaxentTagger(filePath.toString + "/english-left3words/english-left3words-distsim.tagger")